cmd/helpMessages.go

```diff
@@ -134,162 +160,176 @@ const jobsCmdLongDescription = "Sub-commands related to managing jobs."
 
 const jobsCmdExample = "azcopy jobs show [jobID]"
 
-const listJobsCmdShortDescription = "Display information on all jobs"
+const listJobsCmdShortDescription = "Displays information on all jobs"
 
 const listJobsCmdLongDescription = `
-Display information on all jobs.`
+Displays information on all jobs.`
 
 const showJobsCmdShortDescription = "Show detailed information for the given job ID"
 
 const showJobsCmdLongDescription = `
-Show detailed information for the given job ID: if only the job ID is supplied without a flag, then the progress summary of the job is returned.
-(In the progress information shown by this command, the byte counts and percent complete do not include files currently in progress.)
-If the with-status flag is set, then the list of transfers in the job with the given value will be shown.`
+If you provide only a job ID, and not a flag, then this command returns the progress summary only.
+The byte counts and percent complete that appears when you run this command don't reflect files related to jobs that are currently in progress.
+If you set the with-status flag, then only the list of transfers associated with the given status appear.`
```

Comment:
The edit seems to have changed the meaning here.  The byte counts and percent complete are provided for jobs in progress, but they are little innacurate because they only include the files that are _completed_ in the job. They don't not include the _partially completed_ files that are in progress right now.  (Any big live job will usually have some files that are totally completed, some that are inprogress right now, and some that are not started yet.)